config/rootfs/debos/scripts/bullseye-audio.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
```

Comment:
Might it not make sense to get a Debian backport done here if we just need a newer version for newer UCM configs? I can imagine that a bunch of Debian users would benefit and it'd probably be easier to build on top of.

Reply:
Good point, I'll take a look into this.

Reply:
No idea, I don't think anyone had any specific plan there. I'd guess once it gets released at least.